### DISCUSSION TITLE
Plans for a 2.2.3 microrelease before the Ubuntu LTS feature freeze (end of february)?

### DISCUSSION THREAD

**Discussion by Gendra13** [openzfs/zfs#15830] [General]
Hi there

I wanted to ask if there are any planes for a next microrelease (2.2.3) in the near future? 
I am specifically interested because the next Ubuntu LTS version (24.04) is coming nearer and the feature-freeze milestone will be on February, 29th. I think it would be very nice, if the bug-fixes from the last weeks would make it into that release because Ubuntu is quite widely used and the LTS-releases will be out there for many years.

Getting those fixes in before the release seems even more important, when considering the fact, that even critical bugs (like the data corruption one from December) won't be backported into Ubuntu after it is released (so the current LTS 22.04 release is still vulnerable and there is no sign of any fix for it).

**Comment by tonyhutter** [comment]
We're working on a 2.2.3 release: https://github.com/openzfs/zfs/pull/15836

**Comment by robn** [comment]
2.2.3 is now released: https://github.com/openzfs/zfs/releases/tag/zfs-2.2.3

**Reply by Gendra13** [reply]
Yes, I have seen that, thank you.
I already pinged the maintainer for the Ubuntu ZFS package and hopefully this will be included into the upcoming LTS release.
